osc_ingest_trino/trino_utils.py

```diff
@@ -59,23 +55,26 @@ def __call__(self, sqltbl, dbcxn, columns, data_iter):
             self._do_insert(dbcxn, sqltbl, batch)
 
     def _do_insert(self, dbcxn, sqltbl, batch_rows):
-        if self.verbose: print(f'inserting {len(batch_rows)} records')
-        valclause = ',\n'.join(batch_rows)
-        sql = f'insert into {self._full_table_name(sqltbl)} values\n{valclause}'
+        if self.verbose:
+            print(f"inserting {len(batch_rows)} records")
+        valclause = ",\n".join(batch_rows)
+        sql = f"insert into {self._full_table_name(sqltbl)} values\n{valclause}"
         # could add something that prints only summary here, but
         # generally too much data to print reasonably
-        #if self.verbose: print(f'{sql}')
```

Comment:
why the comment spaces?

Reply:
This one is actually a Python standard:
https://peps.python.org/pep-0008/#inline-comments